storagenode/storagenodedb/consoledb.go

```diff
@@ -17,14 +17,15 @@ import (
 )
 
 type consoledb struct {
-	*InfoDB
+	SQLDB
```

Comment:
Wouldnt it make sense to split this into a third database as well, just for consistency?

Reply:
What do you mean? This PR doesn't actually do the splitting of the databases yet (that is a future PR). This PR separates the business objects so they are prepared to run off different database connections.

Or am I misunderstanding the question?